README.md

```diff
@@ -1,54 +1,150 @@
-Code-related documentation can be found in the [documentation](Documentation/) folder.
+**Contributing helps you:**
 
-## Get in touch and participate!
+- Learn browser internals, C++, and rendering engine architecture.
+- Gain hands-on experience in a large-scale open-source project.
+- Help shape an alternative to today's browser engine monopoly.
 
-Join [our Discord server](https://discord.gg/nvfjVJ4Svh) to participate in development discussion.
+---
 
-Please read [Getting started contributing](Documentation/GettingStartedContributing.md) if you plan to contribute to Ladybird for the first time.
+## ✨ Features
 
-Before opening an issue, please see the [issue policy](CONTRIBUTING.md#issue-policy) and the [detailed issue-reporting guidelines](ISSUES.md).
+- Custom rendering engine: [`LibWeb`](https://github.com/SerenityOS/serenity/tree/master/Userland/Libraries/LibWeb)
+- JavaScript engine: `LibJS`
+- WebAssembly via `LibWasm`
+- Audio/Video playback using `LibMedia`
+- Sandboxed, multi-process architecture
+- TLS via `LibTLS`, Unicode via `LibUnicode`
+- Active developer community
 
-The full contribution guidelines can be found in [`CONTRIBUTING.md`](CONTRIBUTING.md).
+---
 
-## License
+## 🧠 Architecture
+
+Ladybird uses a **multi-process design** for robustness and security:
+
+- **UI Process**: Manages the interface and user interactions
+- **WebContent Process**: Renders web pages (1 per tab)
+- **RequestServer Process**: Handles networking
+- **ImageDecoder Process**: Handles image decoding securely
+
+> Many components are shared with SerenityOS, including:
+> - `LibJS`, `LibWeb`, `LibGfx`, `LibHTTP`, `LibCore`, `LibCrypto`, `LibIPC`
+
+---
+
+## 🚀 Quick Start
+
+```bash
+git clone https://github.com/LadybirdBrowser/ladybird.git
+cd ladybird
+cmake -B build
+cmake --build build -j$(nproc)
+./build/Ladybird/Ladybird
+````
+
+> ✅ Make sure your system meets the requirements described in the [Build Instructions](Documentation/BuildInstructionsLadybird.md).
+
+---
+
+## 🔧 Build Instructions
+
+Ladybird builds on:
+
+* Linux
+* macOS
+* Windows (via WSL2)
+
+See full setup: [Documentation/BuildInstructionsLadybird.md](Documentation/BuildInstructionsLadybird.md)
+
+---
+
+## 👋 Getting Started for Web Developers
+
+Coming from a frontend background? Here's how you can help
+
+🧪 Test HTML/CSS/JS rendering**: Spot inconsistencies and report bugs
+🐞 Create minimal repros**: Help isolate rendering bugs with small HTML/CSS test cases
+📄 Improve documentation**: Fix typos, clarify setup, or write guides
+💡 Suggest UI improvements**: Help with visual/UI feedback via Qt
+
+👉 Browse [Good First Issues](https://github.com/LadybirdBrowser/ladybird/issues?q=label%3A%22good+first+issue%22)
+
+
+## 📚 Documentation
+
+Find technical docs and contributor guides in the [Documentation/](Documentation/) directory:
+
+* [Getting Started Contributing](Documentation/GettingStartedContributing.md)
+* [Build Instructions](Documentation/BuildInstructionsLadybird.md)
```

Comment:
make sure you link to the build instructions 50 more times. the past 5 weren't enough